plugin/clone/src/clone_client.cc

```diff
@@ -1072,6 +1083,13 @@ int Client::validate_remote_params() {
     last_error = ER_CLONE_PLUGIN_MATCH;
   }
 
+  /* Donor plugin is not there in donor but installed in recipient. Fail the
+   * clone. */
```

Comment:
Not the MySQL comment style, either `//`, either `/* */` without leading stars in the middle lines